<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slave</class>
 <widget class="QWidget" name="Slave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slave</string>
  </property>
  <widget class="QPushButton" name="run_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>START!!!</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>やる気を出す</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>割と早くなる</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="post_line_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>http://localhost:8080/answer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="get_line_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>http://localhost:8080/problem/problem1.txt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>取得元</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>送信先</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="token_line_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>testman</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>トークン</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prob_num_line_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>問題番号</string>
   </property>
  </widget>
  <widget class="QLabel" name="net_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>レディ</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>run_button</sender>
   <signal>clicked()</signal>
   <receiver>Slave</receiver>
   <slot>clicked_run_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clicked_run_button()</slot>
 </slots>
</ui>
